src/components/App/App.jsx

```diff
@@ -162,6 +162,15 @@ class App extends React.Component {
             return;
         }
 
+        if (index === this.state.selectedParagraph.length - 1) {
+            this.setState({
+                characters,
+                words,
+                timeRemaining: 0,
+            });
+            return;
+        }
+
```

Comment:
This logic looks good, but we should probably also update the score here

Since - if the user finishes the tests before time, the time won't be 60 seconds, and his/her wpm will be more

In case you wish to work on this in this PR itself, it'd be cool

But, If you want, this can be taken up in a separate PR/issue, open another issue and add a comment here with a todo something like this -

```// TODO: Update the test score for this case, here the time taken isn't 60 seconds, so the wpm should be modified```

Reply:
Will the update WPM will be calculated using the below formula?
wpm = wordsTyped * (totalTime/(totalTime - timeRemaining)

For eg:
1)A person types 45 words in 15 sec, his answer will be = 45*60/(60-45) = 180wpm.
2)A person types 20 words in 10 sec, his answer will be = 20*60/(60-40) = 120wpm.

Reply:
Yes, sounds perfect!